src/Map.js

```diff
@@ -294,37 +308,54 @@ define([
          * Holds options for map creation when using [Gp.Map.load()](module-Map.html) function.
          *
          * Availables options are described below.
+         * 
+         * * ** Common 2D/3D properties**
+         * 
+         * | property | Type | Argument | Default | Description |
+         * | - | - | - | - | - |
+         * | apiKey | String / Array.<String> | | | access key(s) to Geoportal platform, obtained [here](http://professionnels.ign.fr/ign/contrats) |
+         * | library | String | optional | "ol3" | Library to load the map with. "ol3" by default (2D) or "itowns" (3D). Use : <ul> <li>GpOL3 bundle to load the map only with openLayers (2D). </li> <li>GpItowns bundle to load the map only with iTowns (3D).</li><li>GpOL3Itowns bundle to be able to switch between ol3 and iTowns views (see {@link Gp.Map#switchToLibITOL3 Gp.Map.switchToLibITOL3()}).</li></ul> |
```

Comment:
Il faudrait que les valeurs possibles soient plus explicites (là on ne sait pas si on doit utiliser "itowns" ou "GpItowns"). De plus, on ne propose pas le bundle GpItowns seul, si ? 
ça donnerait par exemple plutôt (tu peux modifier) : 

Library to load the map with. 
Values are : 
- "ol3" (default) to load the map with OpenLayers (2D) 
- "itowns" to be able to use 2D (OpenLayers) and 3D (iTowns) views. In this case, you have to use GpOl3Itowns.js file (instead of GpOl3.js)

Reply:
Ok je prends :+1: